batch7crud-roll6/src/main/java/com/lftechnology/batch7crud/controller/UserController.java

```diff
@@ -0,0 +1,170 @@
+
+        } catch (DataException e) {
+            e.printStackTrace();
+
+        }
+    }
+
+    private void create(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        request.getRequestDispatcher("/WEB-INF/views/addUser.jsp").forward(request, response);
+    }
+
+    private void createProcess(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+        User user = new User();
+        UserService userService = new UserService();
+
+        String firstName = request.getParameter("firstname");
+        String surName = request.getParameter("surname");
+        String username = request.getParameter("username");
+        String password = request.getParameter("password");
+
+        user.setFirstName(firstName);
+        user.setSurName(surName);
+        user.setUserName(username);
+        user.setPassword(password);
+
+        try {
+            userService.addUser(user);
+            response.sendRedirect("/batch7crud-roll6/users");
+        } catch (DataException e) {
+            System.err.println(e);
+            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+
+        }
+    }
+
+    private void edit(HttpServletRequest request, HttpServletResponse response, int id) throws ServletException, IOException {
+        try {
+            request.setAttribute("user", userService.fetchByID(id));
+            System.out.println("hello im here");
+            request.getRequestDispatcher("/WEB-INF/views/editUser.jsp").forward(request, response);
+        } catch (DataException e) {
+            e.printStackTrace();
+            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+
+        }
+    }
+
+    private void editProcess(HttpServletRequest request, HttpServletResponse response, int id) throws ServletException, IOException {
+
+        User user = new User();
+        UserService userService = new UserService();
+
+        String firstName = request.getParameter("firstname");
+        String surName = request.getParameter("surname");
+        String username = request.getParameter("username");
+
+        user.setFirstName(firstName);
+        user.setSurName(surName);
+        user.setUserName(username);
+        user.setId(id);
+
+        try {
+            userService.update(user);
+            response.sendRedirect("/batch7crud-roll6/users");
+        } catch (DataException e) {
+            System.err.println(e);
+            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+
+        }
+    }
+
+    private void deleteProcess(HttpServletRequest request, HttpServletResponse response, int id) throws ServletException, IOException {
+        try {
+            userService.deleteUser(id);
+            response.sendRedirect("/batch7crud-roll6/users");
+
+        } catch (DataException e) {
+            System.err.println(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)